File: src/main/java/com/lcaohoanq/Spring_Snake_Game/controller/ScoreController.java
package com.lcaohoanq.Spring_Snake_Game.controller;

import com.lcaohoanq.Spring_Snake_Game.model.Score;
import com.lcaohoanq.Spring_Snake_Game.repository.ScoreRepository;
import java.util.List;
import lombok.extern.slf4j.Slf4j;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@Slf4j
@RestController
public class ScoreController {
    @Autowired
    private ScoreRepository scoreRepository;

    @GetMapping("/scores")
    List<Score> all() {
        return scoreRepository.findAll();
    }
}

File: src/main/java/com/lcaohoanq/Spring_Snake_Game/controller/ScoreController.kt
package com.lcaohoanq.Spring_Snake_Game.controller

import com.lcaohoanq.Spring_Snake_Game.entity.Score
import com.lcaohoanq.Spring_Snake_Game.repository.ScoreRepository
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping(path = ["\${v1API}/scores"])
class ScoreController(private val scoreRepository: ScoreRepository) {

    @GetMapping("")
    fun getScore(): List<Score> {
        return scoreRepository.findAll();
    }

}